crates/bevy_render/src/camera/camera.rs

```diff
@@ -308,15 +311,17 @@ impl CameraRenderGraph {
 /// swapchain or an [`Image`].
 #[derive(Debug, Clone, Reflect, PartialEq, Eq, Hash, PartialOrd, Ord)]
 pub enum RenderTarget {
+    /// Renders the camera's view to the initial window.
```

Comment:
This could use a bit more clarity. Do we have module level docs that describe this concept?